api/src/jwt/jwtPermissionMap.spec.ts

```diff
@@ -0,0 +1,24 @@
+import configuration, { Configuration } from "../configuration";
+import { getJwtPermission, parsePermissionMap } from "./jwtPermissionMap";
+
+describe("jwtPremissionMap", () => {
+    let config: Configuration;
+    beforeAll(() => {
+        // This test relies on the .env.test file configured as per .env.test.example
```

Comment:
Perhaps we should instead mock the resulting object here, so the test is not too brittle?